### DISCUSSION TITLE
Proxy speed

### DISCUSSION THREAD

**Discussion by Focl333** [towfiqi/serpbear#227] [Q&A]
Hello guys,
Start to using SerpBear and I like it a lot 

Just questions related to using Proxy as a tool to scrape SERPs for postions.
All of API services I didnt find usefull, so I am trying with proxy option.

**But how its with speed? I bought plan with 100 proxies here:  https://www.webshare.io/pricing**
I upload all of them into Proxy list (100 IPs) but this speed is veeery slow, like 1 keyword / minute.. Is this normal? How can I incrrease speed?

Thank you
Andrej

**Comment by syamsulalam** [comment]
if you need speed perhaps you can try serper.dev it has 4 / 5 concurrent requests at same time each scrapes and takes 15 - 30 sec for each scrapes. so basically 4 keywords / 15 - 30 sec which is better. let's also ask the dev to integrate scrapingbee which has 5 concurrent request also.
